packages/strapi-plugin-sentry/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "strapi-plugin-sentry",
+  "version": "3.3.4",
+  "description": "Send Strapi error events to Sentry",
+  "strapi": {
+    "name": "sentry",
+    "icon": "plug",
+    "description": "Send API errors to Sentry"
```

Comment:
You can use a translation key so the desciption can be changed by the community for every language